tests/e2e/support/api/davSpaces/spaces.ts

```diff
@@ -4,6 +4,36 @@ import join from 'join-path'
 import { getPersonalSpaceId } from '../graph'
 import { config } from '../../../config'
 
+export const doesFolderExists = async ({
+  user,
+  parentFolder,
+  subFolder,
+  oc10Path
+}: {
```

Comment:
or it can only receive path
```suggestion
export const doesFolderExists = async (path)
```